<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorResolver</class>
 <widget class="QMainWindow" name="ErrorResolver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Error Information</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Error Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="errorTypeLabel">
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Item affected: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="itemLabel">
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="descriptionText"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Possible Actions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="removeItem">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>130</height>
          </size>
         </property>
         <property name="text">
          <string>Remove

Item</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="moveUpTimeButton">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>130</height>
          </size>
         </property>
         <property name="text">
          <string>Move Up

Reservation

 End Time</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="timeEditButton">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>130</height>
          </size>
         </property>
         <property name="text">
          <string>Launch Time Editor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
